src/neuroconv/datainterfaces/ecephys/tdt/tdtdatainterface.py

```diff
@@ -17,6 +24,8 @@ def __init__(
             Path to the directory with the corresponding files (TSQ, TBK, TEV, SEV)
         stream_id : str, "0" by default
             Select from multiple streams.
+        gain : float
+            The conversion factor from int16 to Volts.
```

Comment:
```suggestion
            The conversion factor from int16 to microvolts.
```

The `gain` specified here sets the `gain_to_uV` value on the SI recording object, right? So the `gain` we want is `1.0`, but that is to map the `int16` to `uV`, the current problem of which is that it's actually applying to Volts, resulting in the incorrect value of `1e6`

Reply:
Yes, you're right!